### PR TITLE
Add LiveStream component and endpoint

### PR DESCRIPTION
**Pull request by renefs**
- Add endpoint for the live_stream and live_stream_status
- Fix patch method: Using data instead of params for kwargs

### CODE REVIEW

**Comment by renefs** [comment]
Tox is reporting ok and the code formatting is ok as well. Once Travis conf is fixed, the checks should succeed :)

**Comment by prschmid** [comment]
Hi @renefs looks like there are a bunch of failing tests -- Could I trouble you to fix those up and then I'll review the code. Thanks!

**Comment by renefs** [comment]
Done, It seems since my last PR, the develop branch changed a bit, but now It should be ok

File: zoomus/components/live_stream_status.py
Comment (prschmid) [inline zoomus/components/live_stream_status.py:7]: I know that the status is technically separate endpoint on Zoom's side. but do you think for the sake of simplicity in this library that getting a status should be part of the live_stream component. I.e. I would think that doing
`zoom.live_stream.update` and `zoom.live_stream.update_status` might be cleaner/easier? Thoughts?

Comment (renefs) [reply zoomus/components/live_stream_status.py:7]: Hi Patrick, I think It actually makes sense. I think the logic applies to the same element (A Live Stream) and the only difference is the payload and the endpoint. Let me check, I will see what I can do. 

Please don't update the develop branch again 🤪

Comment (prschmid) [reply zoomus/components/live_stream_status.py:7]: Hi! Great! 

I just bumped one library dependency, so please do merge develop in again, but it should be transparent to this work. I'll promise not to make any more merges to develop until this gets merged. in. 

Thank you again!

Comment (renefs) [reply zoomus/components/live_stream_status.py]: Hi Patrick! Changes are done. Just waiting for the merge :) Cheers!

File: .pre-commit-config.yaml
Comment (prschmid) [inline .pre-commit-config.yaml:6]: Was there a reason this was removed?

File: tests/zoomus/test_client.py
Comment (prschmid) [inline tests/zoomus/test_client.py:58]: There looks to be a trailing ',' at the end of this set. This would fail a run of `black`, so please make sure to run `black` to make sure that all the formatting things are taken care of. Thanks!

File: zoomus/components/__init__.py
Comment (prschmid) [inline zoomus/components/__init__.py:6]: If you take a look at what is in develop now, these are now in alphabetical order. When you merge develop back in to this branch, can you please put `live_stream` at the appropriate place? Thanks.

File: tests/zoomus/test_client.py
Comment (prschmid) [inline tests/zoomus/test_client.py:52]: If you take a look at what is in develop now, these are now in alphabetical order. When you merge develop back in to this branch, can you please put `live_stream` at the appropriate place? Thanks.

File: zoomus/client.py
Comment (prschmid) [inline zoomus/client.py:29]: I'm not sure if I made this list alphabetical in develop, but can I trouble you to order this list here and ensure that it is in alphabetical order? Thanks!

**Comment by renefs** [comment]
Apply the changes as you wish

**Comment by prschmid** [comment]
@renefs FYI, I just went ahead and fixed up the merge conflicts. Can you double check if anything else seems amiss and respond to any other questions not pertaining to the merge part and then we can get this merged in today. Thanks for your time and effort!

File: .pre-commit-config.yaml
Comment (renefs) [reply .pre-commit-config.yaml:6]: The precommit was failing for me when using Python 3.7. I think that removing this line, the Python version was not forced. Correct me if I'm wrong though.

Comment (prschmid) [reply .pre-commit-config.yaml:6]: Ahh, got it. We'll want to keep this as it was before since that ensures we're all running it using the same python version. If this is causing you issues on the commit side of things, you could temporarily remove this and run `black` manually to make sure that files are formatted correctly. Then you can add it back in so that it works for everyone else. Thoughts?

File: zoomus/client.py
Comment (renefs) [reply zoomus/client.py]: I think this is already changed

File: zoomus/components/__init__.py
Comment (renefs) [reply zoomus/components/__init__.py]: This is also updated

File: tests/zoomus/test_client.py
Comment (renefs) [reply tests/zoomus/test_client.py]: This is fixed

File: .pre-commit-config.yaml
Comment (renefs) [reply .pre-commit-config.yaml:6]: I'm afraid that I cannot commit if I remove this line

**Comment by jamestiotio** [comment]
Hi @prschmid! May I bump this issue up for merging? The livestream feature is something that I would like to utilize for my research project. Thank you!

**Review by mfldavidson — APPROVED** [review]
Checked out locally and tests passed. Code looks good and will definitely be useful! I think we are ready to merge but I'd like @prschmid to take a final pass before we merge.

**Pull request merged**
